cdci_data_analysis/analysis/parameters.py

```diff
@@ -297,6 +274,10 @@ def set_par(self, value, units=None):
         if units is not None:
             self.units = units
         self.value = value
+        # this should return the value in a default format
```

Comment:
to me, this is the kind of comment which is make entirely redundant by carefully crafted method name :) It almost literally repeats the next line :)
No complains though, just wonder if this is how it also looks for you!

Reply:
Comments add (or at least they should add) clarity and explanations to the code, that's why I add them even though they might effectively be useless or not vital, like in this case. In this specific case, it probably is a leftover comment from a previous commit, just forgot to remove it.

Reply:
comments are critical when the intent is not obvious!
But they are not an excuse for unclear code, when making code clear is a clear option, like it's already done here.